deployment/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 Oracle
```

Comment:
Not trying to be nit-picky, but the copyright check will suggest that the copyright is missing, because it's looking for `Copyright (c) 2021 Oracle and/or its affiliates.` or `Copyright (c) 2020-2021 Oracle and/or its affiliates.` (that format/wording).  Might consider changing it so it doesn't complain about it?